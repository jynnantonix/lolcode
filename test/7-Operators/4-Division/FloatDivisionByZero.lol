This test makes sure that floating point decimal division by 0 raises an error.

HAI
	QUOSHUNT OF 1.0 AN 0.0
KTHXBAI
