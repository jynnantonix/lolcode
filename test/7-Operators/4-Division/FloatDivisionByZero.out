Unexpected error: float division by zero
